### PR TITLE
add cratprotect for testwiki&admin rights add for default

### PR DESCRIPTION
**Pull request by Reception123**

### CODE REVIEW

**Comment by Reception123** [comment]
@Southparkfan is this good now?

File: LocalSettings.php [hunk 1]
```diff
@@ -257,6 +257,12 @@
 				'bot',
 				'sysop',
 				'bureaucrat',
+			'sysop' => array(
+				'autopatrolled',
+				'confirmed',
+				'rollbacker',
+			
+		),
```
Comment (NDKilla) [inline LocalSettings.php]: This needs to be before the sysop, the two arrays look jumbled

File: LocalSettings.php [hunk 2]
```diff
@@ -307,6 +313,11 @@
 			),
 			'wikicreator' => array(
 				'createwiki' => true,
+		'+testwiki' => array(
+			'bureaucrat' => array(
+				'bureaucratprotect' => true,
```
Comment (NDKilla) [inline LocalSettings.php]: This is set up right I think but this right isn't definged or used anywhere

**Comment by Southparkfan** [comment]
What do you want? Restricting edit/move/upload/create to bureaucrats?

**Comment by NDKilla** [comment]
Probably all of the above like I have on my orain wiki and like reception tried to do before. Delete needs to be added as a restriction type and then bureaucrat needs to be given several rights (probbly 'bureaucrat' and 'protect' at the least)

**Comment by Reception123** [comment]
Okay. To make things clear. I want to have bureaucrat protected pages. And the other one is so that administrators can grant rollback,patroll and confirmed

**Comment by Reception123** [comment]
@NDKilla @Southparkfan I have fixed some stuff. Is it fine now?

**Comment by Reception123** [comment]
I will leave it for later.

**Pull request closed without merging**